core/includes/date.inc

```diff
@@ -720,16 +734,19 @@ function date_format_date($date, $type = 'medium', $format = '', $langcode = NUL
  *
  * @see format_interval()
  */
-function date_format_interval($date, $granularity = 2, $display_ago = TRUE) {
+function date_format_interval(?BackdropDateTime $date, $granularity = 2, $display_ago = TRUE) {
```

Comment:
Given, that's it's not really clear, why these empty() checks are done, better remove that type hint (for later investigation).

Reply:
Hm, that type hint (even more removing defensive coding) seems a bit dangerous in a coding standards PR... :thinking: